### DISCUSSION TITLE
I don't see gzip or parsed sizes, it only shows stat size.

### DISCUSSION THREAD

**Discussion by Theoton** [webpack-contrib/webpack-bundle-analyzer#571] [Q&A]
#### Issue description

I don't see gzip or parsed sizes, it only shows stat size.

![image](https://user-images.githubusercontent.com/18453647/84979330-c3043b80-b161-11ea-8c6d-de2276d9ece6.png)
![image](https://user-images.githubusercontent.com/18453647/84979068-145ffb00-b161-11ea-84a5-f33c4ca76367.png)
![image](https://user-images.githubusercontent.com/18453647/84979288-ab2cb780-b161-11ea-98dd-b40c13f7b4ee.png)

#### webpack.config.js

node v11.15.0
webpack  v5.0.0-beta.18

![image](https://user-images.githubusercontent.com/18453647/84979223-83d5ea80-b161-11ea-9d46-23739e17c66d.png)

**Comment by valscion** [comment]
Please create a reproducible project or follow the issue template

**Comment by bfricka** [comment]
This is exact thing happened to me as soon as I added something to a worker file that is also imported by my main thread. This alone enough enough to cause: `Error parsing bundle asset` for every one of the output assets, and 

> No bundles were parsed. Analyzer will show only original module sizes from stats file.

Then none of them can be analyzed via parsed or gzip. Removing this shared dependency from the worker resolves everything. I'm not sure what happens under the hood in [worker-loader](https://github.com/webpack-contrib/worker-loader) that causes this strange behavior. I've had issues in the past with having common modules referenced in workers.

I would expect the worker to simply bundle everything independent of the main bundle. With some shared stuff (TS enums) webpack compilation just hangs, but with other stuff it works fine. In this case I'm pulling in an error reporting SDK. Everything works except for profiling.

**Comment by th0r** [comment]
@Theoton there is not enough information to solve your issue. Could you please create a reproducible example repo or I'll have to close this issue.
